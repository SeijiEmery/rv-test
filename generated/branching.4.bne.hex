0000000    63  14  b5  00  13  03  b0  00  63  14  c5  00  93  03  60  01
0000020    63  14  c5  00  13  0e  10  02  63  14  c5  00  93  0e  c0  02
0000040    73  00  10  00                                                
0000044
