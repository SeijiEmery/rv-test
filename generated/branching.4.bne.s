bne a0, a1, l1
    addi t1, zero, 11
l1:
    bne a0, a2, l2
    addi t2, zero, 22
l2:
    bne a0, a2, l3
    addi t3, zero, 33
l3:
    bne a0, a2, l4
    addi t4, zero, 44
l4:
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
ebreak
