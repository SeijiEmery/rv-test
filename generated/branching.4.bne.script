load /x 0 generated/branching.4.bne.hex
writereg 10 -10
writereg 11 -10
writereg 12 -20
writereg 13 10
writereg 14 20
writereg 6 101
writereg 7 102
writereg 28 103
writereg 29 104
run 0 100000
readreg 13
readreg 14
readreg 6
readreg 7
readreg 28
readreg 29