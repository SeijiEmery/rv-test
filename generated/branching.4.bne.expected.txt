R13 = 10
R14 = 20
R6 = 11
R7 = 102
R28 = 103
R29 = 104
